app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -21,6 +23,7 @@ type DelegationValidationProps = {
   txSuccessTab: string
 }
 
+// REFACTOR: "txSuccessTab", what about using "success notification" instead?
```

Comment:
First, it should be retyped into an enum, currently it's a freestyle string. The name sounds weird but after I checked what its responsibility is, the name quite fits. However, I think it's fundamentally flawed to have such a state variable and it should be changed (it just says what tab most recently sent a successful tx). Currently, if you transfer the funds between two accounts in the account tab, a success message is shown in the Sending tab, which is kinda weird